recipes-bsp/xt-rpi-u-boot-scr/xt-rpi-u-boot-scr.bb

```diff
@@ -16,6 +16,7 @@ SRC_URI = " \
 "
 MMC_PASSTHROUGH_DTBO = "mmc-passthrough.dtbo"
 USB_PASSTHROUGH_DTBO = "usb-passthrough.dtbo"
+WIFI_PASSTHROUGH_DTBO = "wifi-passthrough.dtbo"
```

Comment:
I don't think we need a separate dtbo. It should be configurable

Reply:
This dtbo applied if MACHINE_FEATURES contains domd_wifi

Reply:
I understand. but I see here that we have two configurations:
domd via usb, domd via mmc and domd via wifi but usb and mmc disabled. Is it true?
I see wifi as a optional feature which either can be enabled or disabled. Or if it's mandatory or it is a problem to make it optional then don't add this option

Reply:
It is optional feature. It can be used with rootfs on usb or mmc. Overlays are not applied if MACHINE_FEATURES doesn't contain domd_wifi

Reply:
So why do we have 3 dtbo ? for mmc, usb and wifi?

Reply:
The overlays applied to add features: 
```
usb-passthrough.dtbo + wifi-passthrough.dtbo = USB + NET + WIFI
mmc-passthrough.dtbo + wifi-passthrough.dtbo = MMC + NET + WIFI
mmc-passthrough.dtbo + wifi-passthrough.dtbo + usb-passthrough.dtbo = MMC + USB +NET + WIFI
```

Reply:
Yes. But combination mmc-passthrough.dtbo + wifi-passthrough.dtbo + usb-passthrough.dtbo wasn't tested

Reply:
Of course not, it was a generic answer on generic question "Why we need  dtbo"

Reply:
> So why do we have 3 dtbo ? for mmc, usb and wifi?

Yes

Reply:
Ok thanks